### PR TITLE
Added backend services to Google LB summary.

### PR DESCRIPTION
**Pull request by jtk54**
Added backend services to Google LB summary. @danielpeach PTAL. @duftler FYI.

### CODE REVIEW

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/controllers/GoogleLoadBalancerController.groovy [hunk 1]
```diff
@@ -47,11 +49,24 @@ class GoogleLoadBalancerController {
...
       views.each { GoogleLoadBalancerView view ->
+        def loadBalancerType = view.loadBalancerType
+        def backendServices = []
+        if (loadBalancerType == HTTP) {
+          GoogleHttpLoadBalancer.View httpView = view as GoogleHttpLoadBalancer.View
+          httpView?.hostRules?.each { GoogleHostRule hostRule ->
+            def backendService = null
+            hostRule?.pathMatcher?.pathRules?.each { GooglePathRule pathRule ->
+              backendService = pathRule.backendService
+              backendServices << pathRule.backendService
```
Comment (danielpeach) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/controllers/GoogleLoadBalancerController.groovy]: These should be filtered for duplicates.

**Comment by danielpeach** [comment]
LGTM after comment.

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/controllers/GoogleLoadBalancerController.groovy [hunk 2]
```diff
@@ -47,11 +49,25 @@ class GoogleLoadBalancerController {
...
 
       views.each { GoogleLoadBalancerView view ->
+        def loadBalancerType = view.loadBalancerType
+        def backendServices = []
+        if (loadBalancerType == HTTP) {
+          GoogleHttpLoadBalancer.View httpView = view as GoogleHttpLoadBalancer.View
+          httpView?.hostRules?.each { GoogleHostRule hostRule ->
+            def backendService = null
+            hostRule?.pathMatcher?.pathRules?.each { GooglePathRule pathRule ->
+              backendService = pathRule.backendService
```
Comment (duftler) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/controllers/GoogleLoadBalancerController.groovy]: What is the purpose of this assignment?

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/controllers/GoogleLoadBalancerController.groovy [hunk 3]
```diff
@@ -39,6 +39,8 @@ class GoogleLoadBalancerController {
   @Autowired
   GoogleLoadBalancerProvider googleLoadBalancerProvider
 
+  private static final HTTP = GoogleLoadBalancerType.HTTP.toString()
```
Comment (duftler) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/controllers/GoogleLoadBalancerController.groovy]: Might as well declare this as a `String` here.

**Pull request merged**